_posts/2021-08-13-multi-factor-authentication-with-twilio-verify-api.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Multi Factor Authentication: A Developer’s Perspective on Working with Twilio Verify API"
+slug: "multi-factor-authentication-dev-perspective-on-working-with-twilio-verify-api"
+category: blog
+post_image: /assets/images/multi-factor-authentication/header_image.png
+author: seosamh
+date: 2021-08-13 09:30:00
+excerpt: "Multi Factor Authentication is a method of authentication that adds extra steps of verification for gaining access to resources such as an online account, internal system, application or network. Authentication has been traditionally completed by validating a username and password."
+graphic: /assets/images/multi-factor-authentication/header_image.png
+---
+
+### What is Multi Factor Authentication (MFA)?
```

Comment:
I don't think you actually answered this question below?

What are the factors of authentication? I think either expand a little to describe the multi factors or change question to ask what multi step is?  This is optional, will leave decision to you